### PR TITLE
Alerting: Small readability improvements to template.go

### PR DESCRIPTION
**Pull request by grobinson-grafana**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/ngalert/state/template/template.go [hunk 1]
```diff
@@ -51,59 +49,63 @@ func (v Value) String() string {
 	return strconv.FormatFloat(v.Value, 'f', -1, 64)
 }
 
-func NewValues(values map[string]eval.NumberValueCapture) map[string]Value {
-	m := make(map[string]Value)
-	for k, v := range values {
+func NewValues(caps map[string]eval.NumberValueCapture) map[string]Value {
```
Comment (alexmobo) [inline pkg/services/ngalert/state/template/template.go:52]: Wouldn't it be even more readable if we call it `captures`
```suggestion
func NewValues(captures map[string]eval.NumberValueCapture) map[string]Value {
```

File: pkg/services/ngalert/state/template/template.go [hunk 2]
```diff
@@ -51,59 +49,63 @@ func (v Value) String() string {
...
+	values := make(map[string]Value)
+	for refID, cap := range caps {
```
Comment (alexmobo) [inline pkg/services/ngalert/state/template/template.go:54]: Ditto
```suggestion
	for refID, cap := range captures {
```

**Review by alexmobo — APPROVED** [review]
Small changes. Not critical at all.

File: pkg/services/ngalert/state/template/template.go (hunk 1 above)
Comment (grobinson-grafana) [reply pkg/services/ngalert/state/template/template.go:52]: I'll keep it as `caps` just for now as we use `caps` in the eval package too!

**Pull request merged**